apps/metric_vnode/eqc/metric_vnode_eqc.erl

```diff
@@ -145,15 +142,15 @@ prop_is_empty() ->
                    os:cmd("rm -r data"),
                    os:cmd("mkdir data"),
                    {S, T} = eval(D),
-                   {Empty, _S1} = ?V:is_empty(S),
-                   TreeEmpty = ?T:is_empty(T),
+                   {Empty, _S1} = metric_vnode:is_empty(S),
+                   TreeEmpty = gb_trees:is_empty(T),
                    if
                        Empty == TreeEmpty ->
                            ok;
                        true ->
                            io:format(user, "~p == ~p~n", [S, T])
                    end,
-                   ?V:terminate(normal, S),
+                   metric_vnode:terminate(normal, S),
                    ?WHENFAIL(io:format(user, "L: ~p /= ~p~n", [Empty, TreeEmpty]),
```

Comment:
Line 154 is too long:                    ?WHENFAIL(io:format(user, "L: ~p /= ~p~n", [Empty, TreeEmpty]),.